RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 21.0
 -----
+* [***] Updated About screen for WordPress and Jetpack apps to their respective urls. [https://github.com/wordpress-mobile/WordPress-Android/pull/17282]
```

Comment:
@ravishanker Any reason why this was marked with `[***]`, suggesting it to be a very important and impacting user-facing change?

To me, this change probably doesn't warrant being the most prominent item we want to highlight in our Release Notes (especially for Jetpack where the visible change will be even more minimal), and instead should probably be using the same priority indicator level as the entry for the Splash Screen update that is mentioned just below? Wdyt?

Reply:
Sounds good.  It should be at the same level as Splash Screen.  Thank you.